js/script.js

```diff
@@ -0,0 +1,22 @@
+function getTriangleArea (a, h) {
+  if ((a > 0 ) && (h > 0)) {
+    return (a*h/2);
+  }
+  else {
```

Comment:
else here is unnecessary because return stop function when first if will return true

Reply:
What if someone writes incorrect data ?
I need it to display message that 'Podales nieprawidlowe dane'

Reply:
ok, but you can write it like this:

```
if (something) {
    return some_result;
}
return 'some string with msg';
```